.buildkite/pipeline-cd.yml

```diff
@@ -31,7 +31,7 @@ steps:
 
 - label: 'Deploy android and webgl (linux)'
   command:
-    - ./build.sh -p android,webgl -i -s release debug
+    - ./build.sh -c -p android,webgl -i -s release debug
```

Comment:
Do we know this is needed? Filament makefile dependencies *seem* fairly reliable to me, and will make for much longer build times (but I guess this step is only done when we change filament now right?)

Reply:
Had a chat with @ncoder last friday. He is convinced that this is a good practice. I tend to agree with him, but am a little concerned about the long clean build time.